react-graph/src/LineLegend.tsx

```diff
@@ -58,40 +58,41 @@ function LineLegend(props: IProps) {
     setLabel((props.hasNoData ? Warning : "") + props.label);
   }, [props.hasNoData, props.label]);
 
-  React.useEffect(() => setWLegend(props.size === 'sm' ? context.SmWidth : context.LgWidth), [context.LgWidth, context.SmWidth, props.size]);
-  React.useEffect(() => setHLegend(props.size === 'sm' ? context.SmHeight : context.LgHeight), [context.SmHeight, context.LgHeight, props.size]);
+  React.useEffect(() => setLegendWith(props.size === 'sm' ? context.SmWidth : context.LgWidth), [context.LgWidth, context.SmWidth, props.size]);
+  React.useEffect(() => setLegendHeight(props.size === 'sm' ? context.SmHeight : context.LgHeight), [context.SmHeight, context.LgHeight, props.size]);
 
   React.useEffect(() => {
-    let t = 1;
-    let w = GetTextWidth(textFont, `${t}em`, label);
-    let h = GetTextHeight(textFont, `${t}em`, label);
+    let fontSize = 1;
+    let textWidth = GetTextWidth(textFont, `${fontSize}em`, label, `margin: auto auto auto 0px; display: inline-block; font-weight: 400; font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,"Helvetica Neue",Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";`);
```

Comment:
Should this and below be using the new const for fontFamily?

Reply:
Correct